pic-sure-auth-services/src/main/java/edu/harvard/hms/dbmi/avillach/auth/service/auth/AuthenticationService.java

```diff
@@ -119,17 +120,22 @@ public Response getToken(Map<String, String> authRequest){
 
     private JsonNode retrieveUserInfo(String accessToken){
         String auth0UserInfoURI = JAXRSConfiguration.auth0host + "/userinfo";
-        List<Header> headers = new ArrayList<>();
-        headers.add(new BasicHeader("Content-Type", MediaType.APPLICATION_JSON));
-        headers.add(new BasicHeader("Authorization", "Bearer " + accessToken));
+        Header[] headers = {
```

Comment:
We were making a list, and then pulling the underlying array out of it and never using the list. This seems a bit better.